src/Dev/SSListExporter.php

```diff
@@ -1,85 +0,0 @@
-<?php
```

Comment:
Exporter is now a final class, so this class must be removed

Reply:
Seems like we're not loosing anything

I tested by modifying the expected value in GridFieldDetailFormTest::testEditForm():239

Previously a failure in assertListContains() would look like this:

```
Failed asserting that SilverStripe\ORM\HasManyList Object &0000000000000dc40000000000000000 (
    'dynamicData' => Array &0 ()
    'failover' => null
    'customisedObject' => null
    'objCache' => Array &1 ()
    'extension_instances' => null
    'beforeExtendCallbacks' => Array &2 ()
    'afterExtendCallbacks' => Array &3 ()
    'extra_method_registers' => Array &4 ()
    'dataClass' => 'SilverStripe\Forms\Tests\GridField\GridFieldDetailFormTest\Person'
    'dataQuery' => SilverStripe\ORM\DataQuery Object &0000000000000dc00000000000000000 (
        'dataClass' => 'SilverStripe\Forms\Tests\GridField\GridFieldDetailFormTest\Person'
        'query' => SilverStripe\ORM\Queries\SQLSelect Object &0000000000000dc30000000000000000 (
            'replacementsOld' => Array &5 ()
            'replacementsNew' => Array &6 ()
            'where' => Array &7 (
                0 => Array &8 (
                    '"GridFieldDetailFormTest_Person"."GroupID" = ?' => Array &9 (
                        0 => 1
                    )
                )
            )
            'connective' => 'AND'
            'from' => Array &10 (
                'GridFieldDetailFormTest_Person' => '"GridFieldDetailFormTest_Person"'
            )
            'select' => Array &11 ()
            'groupby' => Array &12 ()
            'having' => Array &13 ()
            'union' => Array &14 ()
            'with' => Array &15 ()
            'distinct' => true
            'orderby' => Array &16 (
                '"FirstName"' => 'ASC'
            )
            'limit' => Array &17 ()
        )
        'collidingFields' => Array &18 ()
        'allowCollidingFieldStatements' => false
        'dataQueryManipulators' => Array &19 ()
        'queriedColumns' => null
        'queryFinalised' => false
        'querySubclasses' => true
        'filterByClassName' => true
        'queryParams' => Array &20 (
            '' => null
            'Foreign.ID' => 1
            'Foreign.Filter' => Array &21 (
                '"GridFieldDetailFormTest_Person"."GroupID"' => 1
            )
        )
        'extension_instances' => null
        'beforeExtendCallbacks' => Array &22 ()
        'afterExtendCallbacks' => Array &23 ()
        'extra_method_registers' => Array &24 ()
    )
    'finalisedQuery' => null
    'eagerLoadRelationChains' => Array &25 ()
    'eagerLoadAllRelations' => Array &26 ()
    'eagerLoadedData' => Array &27 ()
    'inAlterDataQueryCall' => false
    'addCallbacks' => null
    'removeCallbacks' => null
    'foreignKey' => 'GroupID'
)  contains an item matching ( "Surname" is "Bagginsx").
```

Now it looks like this
```
Failed asserting that SilverStripe\ORM\HasManyList Object #2006 (
    'dynamicData' => Array &0 [],
    'failover' => null,
    'customisedObject' => null,
    'objCache' => Array &1 [],
    'extension_instances' => null,
    'beforeExtendCallbacks' => Array &2 [],
    'afterExtendCallbacks' => Array &3 [],
    'extra_method_registers' => Array &4 [],
    'dataClass' => 'SilverStripe\Forms\Tests\GridField\GridFieldDetailFormTest\Person',
    'dataQuery' => SilverStripe\ORM\DataQuery Object #2016 (
        'dataClass' => 'SilverStripe\Forms\Tests\GridField\GridFieldDetailFormTest\Person',
        'query' => SilverStripe\ORM\Queries\SQLSelect Object #2005 (
            'replacementsOld' => Array &5 [],
            'replacementsNew' => Array &6 [],
            'where' => Array &7 [
                0 => Array &8 [
                    '"GridFieldDetailFormTest_Person"."GroupID" = ?' => Array &9 [
                        0 => 1,
                    ],
                ],
            ],
            'connective' => 'AND',
            'from' => Array &10 [
                'GridFieldDetailFormTest_Person' => '"GridFieldDetailFormTest_Person"',
            ],
            'select' => Array &11 [],
            'groupby' => Array &12 [],
            'having' => Array &13 [],
            'union' => Array &14 [],
            'with' => Array &15 [],
            'distinct' => true,
            'orderby' => Array &16 [
                '"FirstName"' => 'ASC',
            ],
            'limit' => Array &17 [],
        ),
        'collidingFields' => Array &18 [],
        'allowCollidingFieldStatements' => false,
        'dataQueryManipulators' => Array &19 [],
        'queriedColumns' => null,
        'queryFinalised' => false,
        'querySubclasses' => true,
        'filterByClassName' => true,
        'queryParams' => Array &20 [
            '' => null,
            'Foreign.ID' => 1,
            'Foreign.Filter' => Array &21 [
                '"GridFieldDetailFormTest_Person"."GroupID"' => 1,
            ],
        ],
        'extension_instances' => null,
        'beforeExtendCallbacks' => Array &22 [],
        'afterExtendCallbacks' => Array &23 [],
        'extra_method_registers' => Array &24 [],
    ),
    'finalisedQuery' => null,
    'eagerLoadRelationChains' => Array &25 [],
    'eagerLoadAllRelations' => Array &26 [],
    'eagerLoadedData' => Array &27 [],
    'inAlterDataQueryCall' => false,
    'addCallbacks' => null,
    'removeCallbacks' => null,
    'foreignKey' => 'GroupID',
)  contains an item matching ( "Surname" is "Bagginsx").
```